release-process-documentation/release-team-guides/ci-signal-playbook.md

```diff
@@ -0,0 +1,77 @@
+# CI Signal Lead Playbook
+
+TODO: This could be worded / structured better but I want it to live someplace more "real" than [a PR comment](https://github.com/kubernetes/test-infra/pull/5306#issuecomment-341277823)
+
+## Status Quo
+
+Currently, here's what I'm doing as the CI Signal role for release 1.9:
+
+### Checking [sig-release-master-blocking](https://k8s-testgrid.appspot.com/sig-release-master-blocking) daily
+
+- if a job is failing in one of the meta-stages (Up, Down, DumpClusterLogs, etc), I find the owning sig
+- if a job is failing because a specific test case is failing, and that test case has a [sig-foo] in its name, I go bug sig-foo
+- with unit test case failures, I try to infer the sig from the path, or OWNERS files in it, otherwise I find the owning sig to help
+- with verify failures, I try to infer the failure from the log, otherwise I find the owning sig to help
+- if a test case is failing in one job consistently, but not others, both the job owner and test case owner are responsible for identifying why this combination is different
+
+### Pinging SIG's
+
+I do this by using @kubernetes/sig-foo-test-failures teams when I file issues, or going to the #sig-foo slack channel if I need routing help and know someone's around
+
+If a job as a whole is failing, I file an issue in kubernetes/kubernetes titled:`[job failure] job name issue`
```

Comment:
Are issues not included in the milestone?

Reply:
... good point, they should be, and I don't currently have the power to add issues to the milestone

I have gotten by in interim by bugging someone else on the team with milestone powers, but if we can't get prow or something to add milestone in time, I should ask for the privs.

Reply:
Opened https://github.com/kubernetes/test-infra/issues/5344